apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -31,5 +31,24 @@ fun <T : Path> Expect<T>.exists(): Expect<T> = addAssertion(ExpectImpl.path.exis
  */
 fun <T : Path> Expect<T>.existsNot(): Expect<T> = addAssertion(ExpectImpl.path.existsNot(this))
 
+/**
+ * Creates an [Expect] for the property [Path.getParent] of the subject of the assertion,
+ * so that further fluent calls are assertions about it.
```

Comment:
mention that it fails in case it does not have a parent. I would change the text to something like
```
/*
 * Expects that this [Path] has a [parent][Path.getParent] and creates an [Expect] for it, 
 * so that further fluent calls are assertions about it.
 */
```

Reply:
done